Security/Authorization/Interception/MethodSecurityInterceptor.php

```diff
@@ -41,20 +39,51 @@
  */
 class MethodSecurityInterceptor implements MethodInterceptorInterface
 {
+    /** @var bool */
     private $alwaysAuthenticate;
-    private $securityContext;
+
+    /** @var TokenStorageInterface */
+    private $tokenStorage;
+
+    /** @var MetadataFactoryInterface */
     private $metadataFactory;
+
+    /** @var AuthenticationManagerInterface */
     private $authenticationManager;
+
+    /** @var AccessDecisionManagerInterface */
     private $accessDecisionManager;
+
+    /** @var AfterInvocationManagerInterface */
     private $afterInvocationManager;
+
+    /** @var RunAsManagerInterface */
     private $runAsManager;
+
+    /** @var LoggerInterface */
     private $logger;
 
-    public function __construct(SecurityContextInterface $securityContext, AuthenticationManagerInterface $authenticationManager, AccessDecisionManagerInterface $accessDecisionManager,
-                                AfterInvocationManagerInterface $afterInvocationManager, RunAsManagerInterface $runAsManager, MetadataFactoryInterface $metadataFactory, LoggerInterface $logger = null)
-    {
+    /**
+     * MethodSecurityInterceptor constructor.
```

Comment:
this comment does not bring any benefit (actually, none of the phpdoc does as all arguments are typehinted)